src/pages/Judging/index.js

```diff
@@ -69,13 +71,21 @@ const getProjects = async userId => {
 
 export default () => {
   const [isJudgingOpen, setIsJudgingOpen] = useState()
+  const [isBlocked, setIsBlocked] = useState()
   const { user } = useAuth()
   const [projects, setProjects] = useState([])
 
   useEffect(() => {
     ;(async () => setProjects(await getProjects(user.uid)))()
   }, [setProjects, user.uid])
 
+  useEffect(() => {
+    ;(async () => {
+      const applicantData = (await applicantsRef.doc(user.uid).get()).data()
+      setIsBlocked(!!!applicantData.submittedProject)
```

Comment:
i think ! === !!!

Reply:
you've seen `!`, now get ready for `!!!!!`